src/lib/import-export/export/exporters/default-exporter.ts

```diff
@@ -166,8 +166,30 @@ export class DefaultExporter extends EventEmitter implements Exporter {
 
 			if ( partialFolderItems ) {
 				for ( const itemName of partialFolderItems ) {
-					const itemPath = path.join( absolutePath, itemName );
-					const itemArchivePath = path.join( archivePath, itemName );
+					// For muPlugins and fonts, paths in contents array may be relative to wp-content, not the category folder
+					// e.g., "index.php" (wp-content root), "mu-plugins/file.php", "fonts/custom.woff"
+					let itemPath: string;
+					let itemArchivePath: string;
+
+					if ( category === 'muPlugins' || category === 'fonts' ) {
+						const isRootFile = ! itemName.includes( '/' );
+						const belongsToCategory =
+							itemName.startsWith( `${ folderName }/` ) ||
+							itemName === folderName ||
+							( category === 'muPlugins' &&
+								( itemName.startsWith( 'languages/' ) || itemName === 'languages' ) ) ||
+							isRootFile;
+
+						if ( belongsToCategory ) {
+							itemPath = path.join( this.options.site.path, 'wp-content', itemName );
+							itemArchivePath = path.join( 'wp-content', itemName );
+						} else {
+							continue;
+						}
```

Comment:
So if the file lives directly inside `wp-content`, we don't add it to the archive..? Shouldn't we exclude it from the modal tree view in the first place, then?